### PR TITLE
Manage public-api keys

### PR DESCRIPTION
**Pull request by Mikescops**
As part of the new Public API efforts, here are new commands to manage API keys.

### CODE REVIEW

File: src/commands/team/publicAPI.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { createPublicAPIKeyHandler, listPublicAPIKeysHandler, revokePublicAPIKeyHandler } from '../../command-handlers';
+import { logger } from '../../logger.js';
+import { Command } from 'commander';
+
+export const PublicAPICommands = (params: { teamGroup: Command }) => {
```
Comment (showi) [inline src/commands/team/publicAPI.ts:5]: np: is that a cap rebellion or smth?

File: src/endpoints/listPublicAPIKeys.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        /**
+         * The access key
+         */
+        accessKey: string;
+        /**
+         * The description of the api key
+         */
+        description: string;
+        /**
+         * Platform used to create the report api key
```
Comment (showi) [inline src/endpoints/listPublicAPIKeys.ts:34]: isnt't it just public api same just after ?

**Review by showi — APPROVED** [review]

File: src/commands/team/publicAPI.ts (hunk 1 above)
Comment (Mikescops) [reply src/commands/team/publicAPI.ts:5]: lol i missed some renames ahah

File: src/endpoints/listPublicAPIKeys.ts (hunk 2 above)
Comment (Mikescops) [reply src/endpoints/listPublicAPIKeys.ts]: yeah forgot to rename

**Pull request merged**